### PR TITLE
Split dbt web configs

### PR DESCRIPTION
**Pull request by rlh1994**
Split out config pages to be clearer about the type of variables we have to make it easier to find what you're looking for.

Starting with web then once the sections are agreed I'll do the rest. I don't particularly like the third section's name.

### CODE REVIEW

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 1]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| `derived_tstamp_partitioned` | Boolean to enable filtering the events table on `derived_tstamp` in addition to `collector_tstamp` (BigQuery only).                                                                                                                                                                                                                                                                                                                  | `true`                                 |
+| `dev_target_name`            | The [target name](https://docs.getdbt.com/reference/profiles.yml) of your development environment as defined in your `profiles.yml` file. See the 'Manifest Tables' section for more details.                                                                                                                                                                                                                                        | `dev`                                  |
```
Comment (georgewoodhead) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:28]:
```suggestion
| `dev_target_name`            | The [target name](https://docs.getdbt.com/reference/profiles.yml) of your development environment as defined in your `profiles.yml` file. See the [Manifest Tables](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-operation/#manifest-tables) section for more details.                                                                                                                                                                                                                                        | `dev`                                  |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 2]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+
+### Operation and logic
+| Variable Name                 | Description                                                                                                                                                                                                                                                                                  | Default        |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| `allow_refresh`               | Used as the default value to return from the `allow_refresh()` macro. This macro determines whether the manifest tables can be refreshed or not, depending on your environment. See the 'Manifest Tables' section for more details.                                                          | `false`        |
```
Comment (georgewoodhead) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:36]:
```suggestion
| `allow_refresh`               | Used as the default value to return from the `allow_refresh()` macro. This macro determines whether the manifest tables can be refreshed or not, depending on your environment. See the [Manifest Tables](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-operation/#manifest-tables) section for more details.                                                          | `false`        |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 3]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| `database`                   | The database that contains your atomic events table.                                                                                                                                                                                                                                                                                                                                                                                 | `target.database`                      |
+| `derived_tstamp_partitioned` | Boolean to enable filtering the events table on `derived_tstamp` in addition to `collector_tstamp` (BigQuery only).                                                                                                                                                                                                                                                                                                                  | `true`                                 |
```
Comment (georgewoodhead) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:27]: Should we move this one to the 'Warehouse Specific' section?

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 4]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| `lookback_window_hours`       | The number of hours to look before the latest event processed - to account for late arriving data, which comes out of order.                                                                                                                                                                 | `6`              |
+| `max_session_days`            | The maximum allowed session length in days. For a session exceeding this length, all events after this limit will stop being processed. Exists to reduce lengthy table scans that can occur due to long sessions which are usually a result of bots.                                         | `3`              |
+| `query_tag`                   | This sets the value of the query_tag for Snowflake database use. This is used internally for metric gathering in Snowflake and its value should not be changed.                                                                                                                              | `snowplow_dbt` |
```
Comment (georgewoodhead) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:42]: Thoughts on moving this one  to Warehouse specific too?

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 5]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| `session_lookback_days`       | Number of days to limit scan on `snowplow_web_base_sessions_lifecycle_manifest` manifest. Exists to improve performance of model when we have a lot of sessions. Should be set to as large a number as practical.                                                                            | `730`            |
+| `session_stitching`           | Determines whether to apply the user mapping to the sessions table. Please see the 'User Mapping' section for more details.                                                                                                                                                                  | `true`         |
```
Comment (georgewoodhead) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:44]:
```suggestion
| `session_stitching`           | Determines whether to apply the user mapping to the sessions table. Please see the [User Mapping](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-user-mapping/) section for more details.                                                                                                                                                                  | `true`         |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 6]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| `start_date`                  | The date to start processing events from in the package, based on `collector_tstamp`.                                                                                                                                                                                                        | `'2020-01-01'`   |
+| `upsert_lookback_days`        | Number of day to look back over the incremental derived tables during the upsert. Where performance is not a concern, should be set to as long a value as possible. Having too short a period can result in duplicates. Please see the incremental materialization section for more details. | `30`             |
```
Comment (georgewoodhead) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:46]:
```suggestion
| `upsert_lookback_days`        | Number of days to look back over the incremental derived tables during the upsert. Where performance is not a concern, should be set to as long a value as possible. Having too short a period can result in duplicates. Please see the [Snowplow Optimized Materialization](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-incremental-materialization/) section for more details. | `30`             |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 7]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| `enable_yauaa`       | Flag to include the [YAUAA enrichment](/docs/enriching-your-data/available-enrichments/yauaa-enrichment/index.md) columns in the models.                                                                                                                                                       | `false` |
+| `has_log_enabled`    | When executed, the package logs information about the current run to the CLI. This can be disabled by setting to `false`.                                                                                                                                                                      | `true`  |
+| `ua_bot_filter`      | Flag to filter out bots via the useragent string pattern match.                                                                                                                                                                                                                                | `true`  |
+
+### Warehouse Specific 
+
+<Tabs groupId="warehouse" queryString>
+<TabItem value="databricks" label="databricks" default>
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:61]:
```suggestion
<TabItem value="databricks" label="Databricks" default>
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 8]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+
+| Variable Name        | Description                                                                                                                                                                                                                                                                                          | Default          |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------- |
+| `databricks_catalog` | The catalogue your atomic events table is in. Depending on the use case it should either be the catalog (for Unity Catalog users from databricks connector 1.1.1 onwards, defaulted to `hive_metastore`) or the same value as your `snowplow__atomic_schema` (unless changed it should be 'atomic'). | `hive_metastore` |
+
+</TabItem>
+<TabItem value="redshift+postgres" label="redshift+postgres">
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:68]:
```suggestion
<TabItem value="redshift+postgres" label="Redshift & Postgres">
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 9]
```diff
@@ -3,45 +3,83 @@ title: "Web"
...
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `enable_iab`         | Flag to include the [IAB enrichment](/docs/enriching-your-data/available-enrichments/iab-enrichment/index.md) columns in the models.                                                                                                                                                           | `false` |
+| `enable_load_tstamp` | Flag to include the `load_tstamp` column in the redshift and postgres base events this run models. This should be set to true (the default) unless you are using the postgres loader or an RDB loader version less than 4.0.0. It must be true to use consent models on postgres and redshift. | `true`  |
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:52]:
```suggestion
| `enable_load_tstamp` | Flag to explicitly include the `load_tstamp` column in the Redshift and Postgres base events this run models. This should be set to true (the default) unless you are using the Postgres loader or an RDB loader version less than 4.0.0. It must be true to use consent models on Postgres and Redshift. | `true`  |
```

**Review by emielver — COMMENTED** [review]
LGTM, have left some slight nitpicky changes

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 10]
```diff
@@ -3,45 +3,101 @@ title: "Web"
...
+| Variable Name                | Description                                                                                         | Default |
+| ---------------------------- | --------------------------------------------------------------------------------------------------- | ------- |
+| `derived_tstamp_partitioned` | Boolean to enable filtering the events table on `derived_tstamp` in addition to `collector_tstamp`. | `true`  |
+
+</TabItem>
+<TabItem value="snowflake" label="Snowflake" default>
+
+| Variable Name | Description                                                                                                                                                                    | Default        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- |
+| `query_tag`   | This sets the value of the query_tag for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:97]:
```suggestion
| `query_tag`   | This sets the value of the `query_tag` for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/normalize/index.md [hunk 11]
```diff
@@ -12,22 +18,54 @@ All variables in Snowplow packages start with `snowplow__` but we have removed t
...
+| Variable Name                | Description                                                                                         | Default |
+| ---------------------------- | --------------------------------------------------------------------------------------------------- | ------- |
+| `derived_tstamp_partitioned` | Boolean to enable filtering the events table on `derived_tstamp` in addition to `collector_tstamp`. | `true`  |
+
+</TabItem>
+<TabItem value="snowflake" label="Snowflake" default>
+
+| Variable Name | Description                                                                                                                                                                    | Default        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- |
+| `query_tag`   | This sets the value of the query_tag for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/normalize/index.md:65]:
```suggestion
| `query_tag`   | This sets the value of the `query_tag` for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/ecommerce/index.md [hunk 12]
```diff
@@ -14,38 +20,67 @@ All variables in Snowplow packages start with `snowplow__` but we have removed t
...
+| Variable Name                | Description                                                                                         | Default |
+| ---------------------------- | --------------------------------------------------------------------------------------------------- | ------- |
+| `derived_tstamp_partitioned` | Boolean to enable filtering the events table on `derived_tstamp` in addition to `collector_tstamp`. | `true`  |
+
+</TabItem>
+<TabItem value="snowflake" label="Snowflake" default>
+
+| Variable Name | Description                                                                                                                                                                    | Default        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- |
+| `query_tag`   | This sets the value of the query_tag for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/ecommerce/index.md:80]:
```suggestion
| `query_tag`   | This sets the value of the `query_tag` for all sql executed against the database. This is used internally for metric gathering in Snowflake and its value should not be changed. | `snowplow_dbt` |
```

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 13]
```diff
@@ -3,45 +3,101 @@ title: "Web"
 sidebar_position: 101
 ---
 
-## Model Configuration
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Package Configuration Variables
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:11]: Why did we change this one but not the rest?

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 14]
```diff
@@ -3,45 +3,101 @@ title: "Web"
...
 
 This packages make use of a series of other variables, which are all set to the recommend values for the operation of the models. Depending on your use case, you might want to override these values by adding to your `dbt_project.yml` file.
```
Comment (emielver) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:13]:
```suggestion
This package utilizes a set of variables that are configured to recommended values for optimal performance of the models. Depending on your use case, you might want to override these values by adding to your `dbt_project.yml` file.
```

Comment (emielver) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:13]: Consider changing for all if you're happy with this suggestion

**Review by emielver — APPROVED** [review]
Just some small changes but all good otherwise I think!

**Pull request merged**